grafana/provisioning/dashboards/CHT/cht_admin_details.json

```diff
@@ -829,7 +829,7 @@
                 "uid": "PBFA97CFB590B2093"
               },
               "editorMode": "code",
-              "expr": "rate(cht_couchdb_update_sequence{db=\"medic\"}[30d]) * 60 * 60",
+              "expr": "rate(cht_couchdb_update_sequence{db=\"medic\", instance=~\"$cht_instance\"}[30d]) * 60 * 60 + 500",
```

Comment:
Sorry, should have explained the rational there!  I added the buffer here because this should match the threshold for the Sentinel Backlog alert rule (which also has the `+500` buffer).  I realized when I went in to fix this that I did not have the buffer for the panel threshold.

As far as the general purpose of the buffer (in both the alert rule and the panel), the idea is to provide a minimum level for the threshold, so that small instances do not issue frivolous warnings about very small Sentinel backlogs.

Reply:
Makes sense, thanks for the explanation!